packages/core/src/report/generate-report.ts

```diff
@@ -131,10 +132,23 @@ export async function generateReport(
   const suggestionInput = { arboristValues, rootArboristNode };
 
   for (const plugin of processedPlugins) {
-    if (plugin.getSuggestions) {
-      const suggestions = await plugin?.getSuggestions(suggestionInput);
-
-      report.suggestions.push(...suggestions);
+    if (plugin.getTasks) {
+      const suggestionTasks = plugin?.getTasks();
```

Comment:
shouldn't need `?` here because you are checking the existence of `getTasks` in the if check
```suggestion
      const suggestionTasks = plugin.getTasks();
```